Makefile

```diff
@@ -89,7 +89,7 @@ oldtest: | nvim helptags
 	+$(SINGLE_MAKE) -C src/nvim/testdir $(MAKEOVERRIDES)
 
 helptags: | nvim
-	+$(BUILD_CMD) -C build runtime/doc/tags
```

Comment:
Hm.. I guess I was just confused because I didn't see any output from running it. Also, `make clean && cd build && make runtime/doc/tags` doesn't work, while `make clean && cd build && make helptags` does.. although that dependency is already contained in the top-level Makefile.